plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/CreateQueryGroupAction.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm;
```

Comment:
Can you move this class to action package within wlm? It will look like below:

```
package org.opensearch.plugin.wlm.action;
```

This will be consistent with how we define actions in the core as well